### PR TITLE
Fix doctests

### PR DESCRIPTION
**Pull request by leotrs**
Currently the doctests are broken. This fixes them. Thanks @kolibril13  for reporting.

Depends on #796

### CODE REVIEW

**Comment by leotrs** [comment]
For future reference, the doctests can be ran by executing `make doctests`. We should add this to our CI...

File: docs/source/tutorials/configuration.rst [hunk 1]
```diff
@@ -335,24 +335,23 @@ A list of all config options
...
-   'custom_folders', 'disable_caching', 'dry_run', 'ffmpeg_loglevel',
-   'flush_cache', 'frame_height', 'frame_rate', 'frame_size', 'frame_width',
-   'frame_x_radius', 'frame_y_radius', 'from_animation_number', 'images_dir',
-   'input_file', 'js_renderer_path', 'leave_progress_bars', 'left_side',
-   'log_dir', 'log_to_file', 'max_files_cached', 'media_dir',
-   'movie_file_extension', 'output_file', 'partial_movie_dir', 'pixel_height',
-   'pixel_width', 'png_mode', 'preview', 'progress_bar', 'quality',
-   'right_side', 'save_as_gif', 'save_last_frame', 'save_pngs', 'scene_names',
-   'show_in_file_browser', 'skip_animations', 'sound', 'tex_dir',
+   ['aspect_ratio', 'assets_dir', 'background_color', 'background_opacity', 'bottom',
```
Comment (kolibril13) [inline docs/source/tutorials/configuration.rst:343]: Nice, assets_dir is now there! :)

**Review by kolibril13 — APPROVED** [review]

**Comment by behackl** [comment]
Lets enable this in our pipeline: could you add the lines

```
      - name: Run doctests in rst files
        run: cd docs && poetry run make doctest
```

to the end of our `ci.yml` file?

**Comment by leotrs** [comment]
@behackl thanks, just did. Note after #796 , we should add `cd docs && poetry run make doctest O=-tskip-manim` instead. So perhaps we can wait until #796 is merged to add this one.

**Comment by leotrs** [comment]
Yeah, right now all our tests are stuck on building every scene in the docs. Two things to do:

+ leave as is, because in this way we will know if all example scenes render correctly in all platforms (though this will multiply our CI time considerably)
+ wait until #796 is merged so we can run doctests without rendering scenes.

**Comment by behackl** [comment]
> * wait until #796 is merged so we can run doctests without rendering scenes.

Definitely this option. The struggle of getting RTD to build our docs is enough.

**Comment by leotrs** [comment]
Depends on #796

**Comment by behackl** [comment]
Doctest under Windows only seem to work when calling `result = subprocess.run(['manim', '-h'], shell=True, stdout=subprocess.PIPE)` in the doctest in `configuration.rst`, and under Ubuntu and MacOS they only seem to work with `result = subprocess.run(['manim', '-h'], stdout=subprocess.PIPE)`. Any ideas how to treat this?

(Otherwise this works really well; the documentation is built successfully without rendering videos.)

**Comment by behackl** [comment]
Doctest under Windows only seem to work when calling `result = subprocess.run(['manim', '-h'], shell=True, stdout=subprocess.PIPE)` in the doctest in `configuration.rst`, and under Ubuntu and MacOS they only seem to work with `result = subprocess.run(['manim', '-h'], stdout=subprocess.PIPE)`. Any ideas how to treat this?

(Otherwise this works really well; the documentation is built successfully without rendering videos.)

**Comment by behackl** [comment]
Doctest under Windows only seem to work when calling `result = subprocess.run(['manim', '-h'], shell=True, stdout=subprocess.PIPE)` in the doctest in `configuration.rst`, and under Ubuntu and MacOS they only seem to work with `result = subprocess.run(['manim', '-h'], stdout=subprocess.PIPE)`. Any ideas how to treat this?

(Otherwise this works really well; the documentation is built successfully without rendering videos.)

**Comment by leotrs** [comment]
There's all sorts of problems with executing shell commands on windows. See e.g. [this](https://docs.python.org/3/library/subprocess.html#frequently-used-arguments) and [this](https://stackoverflow.com/questions/3172470/actual-meaning-of-shell-true-in-subprocess).

I vote we just do something like `shell=True if on_windows else False` and call it a day :sweat_smile:

**Review by behackl — APPROVED** [review]
Everything here is fixed, good to go. Thanks for fixing the doctests!

**Pull request merged**